onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -114,12 +114,12 @@ def testRunModelSymbolicInput(self):
         self.assertEqual(input_name, "X")
         input_shape = sess.get_inputs()[0].shape
         # Input X has an unknown dimension.
-        self.assertEqual(input_shape, [None, 2])
+        self.assertEqual(input_shape, ['None', 2])
```

Comment:
Without the special case treatment, the output now has the dim_param value in quotes. The previous output was the python None for a null.